src/csharp/Grpc.Tools.Tests/Grpc.Tools.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
```

Comment:
please add 
```
<Import Project="..\Grpc.Core\Version.csproj.include" />
<Import Project="..\Grpc.Core\Common.csproj.include" />
```
- the Common.csproj.include should fix the linux build problem
- we want all csproj files to be consistent.

Reply:
Undoable, references Grpc.Core.dll.